snapshot/src/main/java/io/camunda/zeebe/snapshots/impl/FileBasedSnapshotStore.java

```diff
@@ -341,8 +361,8 @@ public ActorFuture<Void> copySnapshot(
 
   @Override
   public FileBasedReceivedSnapshot newReceivedSnapshot(final String snapshotId) {
-    final var optMetadata = FileBasedSnapshotMetadata.ofFileName(snapshotId);
-    final var metadata =
+    final var optMetadata = FileBasedSnapshotId.ofFileName(snapshotId);
```

Comment:
:wrench: variable name should be adjusted. There are more cases of this than this one, maybe it's worth to go over all usages of `FileBasedSnapshotId`